api/src/types/validation.ts

```diff
@@ -24,6 +24,7 @@ const s2paperId = Joi.string().pattern(/[a-f0-9]{40}/);
  * https://arxiv.org/help/arxiv_identifier.
  */
 const currentArxivFormat = Joi.string().pattern(/arxiv:[0-9]{2}[0-9]{2}.[0-9]+(v[0-9]+)?/);
+const arxivMathFormat = Joi.string().pattern(/arxiv:math\/[0-9]{5,}(v[0-9]+)/);
```

Comment:
I think there are arXiv ids in a similar format that start with other things apart from 'math' (looking at this https://arxiv.org/help/arxiv_identifier). (Not objecting to this addition, more of an FYI.)

Reply:
Oh! So it looks like this is actually a bug in the existing `olderArxivFormat` regex where it's assuming that optional bit after the subject (`.GT`) is always present.